Source/WebGPU/WGSL/Parser.cpp

```diff
@@ -329,6 +332,45 @@ Expected<UniqueRef<AST::TypeDecl>, Error> Parser<Lexer>::parseTypeDeclAfterIdent
     RETURN_NODE_REF(NamedType, WTFMove(name));
 }
 
+// https://www.w3.org/TR/WGSL/#syntax-array_type_decl
+// ArrayTypeDecl:
```

Comment:
I was doing this same practice in the WebGPU code, but then people got mad at me so I ended up deleting them. https://github.com/WebKit/WebKit/commit/1c96679bb04a4af96c04cb275c071b2b1c04f9ac

The rationale is that comments should answer "why" not "what". The "what" is already described by the code.

Reply:
Ok. Personally, I like to have the grammar that it's trying to parse there, so I don't have to go the spec to look it up. But, code comments can get out-of-sync with the ground truth.

Reply:
Yeah, spec links are valuable.